tools/auto_analyse_raw_data.py

```diff
@@ -765,7 +765,13 @@ def main():
   add_rawdata_args(arg_parser)
   arg_options = arg_parser.parse_args()
 
-  parse_and_report(get_rawdata(arg_options), arg_options.margin,
+  raw_data = get_rawdata(arg_options)
+  if len(raw_data) == 0:
```

Comment:
Nitpick:
 I think the correct Python idiom is:
`if not len(raw_data):`

Reply:
thanks! will update.
Python isn't my best lang, but C is even worse :D

Reply:
Hmm
`auto_analyse_raw_data.py:769:5: C1801: Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty (len-as-condition)`

so it should be `if not raw_data:`
in testing this I found a different issue as well so a `.strip()` was added

Reply:
Trust the linter. Not me. :)
I knew it was "not right", but clearly I didn't know the "right" answer either.

Reply:
I totally missed running the linter (different machine issues)
But hopefully good now.